extra-multimedia/feeluown/autobuild/beyond

```diff
@@ -11,7 +11,7 @@ Terminal=false
 StartupNotify=true
 EOF
 
-rm -f "$PKGDIR"/usr/bin/feeluown-{install-dev,genicon,update}
+rm -f "$PKGDIR"/usr/bin/feeluown-genicon
```

Comment:
In custom Autobuild scripts, operations should be *verbose by default* (i.e. `rm -fv` instead of `rm -f`), and be marked with abinfo in *each step* (i.e. `abinfo "Removing feeluown-genicon ..."`).

See the "Scripting" section in the [AOSC OS Package Styling Manual](https://wiki.aosc.io/developer/packaging/package-styling-manual/#scripting).

Reply:
There are plenty of good examples in our tree.